src/main/scala/projectEuler/Application.scala

```diff
@@ -12,6 +12,11 @@ object Application {
     println("Problem 2. By considering the terms in the Fibonacci sequence whose values do not exceed four million, find the sum of the even-valued terms.")
     println("Result: " + fibonacciCalculator.sumOfEvenValues)
 
+    println("*-*-*--*-*-*-*-*-* *-*-*--*-*-*-*-*-*")
+    val largestPrimeFactor = new LargestPrimeFactor(600851475143L)
```

Comment:
It is always best to avoid repeating yourself (such as the prime factor number), so if you pull out the value into a variable, then you can use string interpolation to avoid copy and pasting this long number